src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -929,6 +964,17 @@ public function getCasts()
         return $this->casts;
     }
 
+    /**
+     * Get the cast from casts array.
+     *
+     * @param  string  $key
+     * @return mixed
```

Comment:
```suggestion
     * @return string|null
```
I don't think anything else can be returned here?

Reply:
Also I think.